accessibilityTest/AccessibilityDynamicHierarchicalFacet.ts

```diff
@@ -10,7 +10,7 @@ export const AccessibilityDynamicHierarchicalFacet = () => {
       dynamicHierarchicalFacet = $$('div', {
         className: Component.computeCssClassName(DynamicHierarchicalFacet),
         dataTitle: 'My Facet',
-        dataField: '@objecttype',
+        dataField: '@champion_categories',
```

Comment:
Out of curiosity, why's needed?

Reply:
A hierarchical facet works better with fields that have hierarchical values. Like on an ecommerce website: categories and sub-categories for products.
Here the `@champion_categories` field is such a field which makes it better for the test. The `@objecttype` field was on a single level.

Reply:
Thanks